packages/survey-creator-core/src/components/tabs/test-plugin.ts

```diff
@@ -190,6 +195,59 @@ export class TabTestPlugin implements ICreatorPlugin {
       items.push(this.invisibleToggleAction);
     }
 
+    const getThemeTitle = name => this.creator.getLocString("ed." + name + "Theme");
+    const themeMapper = [
+      { name: "defaultV2", title: getThemeTitle("defaultV2"), theme: defaultV2Css },
+      { name: "modern", title: getThemeTitle("modern"), theme: modernCss },
+      { name: "default", title: getThemeTitle("default"), theme: defaultStandardCss }
+    ];
+    const styles = getComputedStyle(document.body);
+    const availableThemes = themeMapper
+      .filter(item => styles.getPropertyValue(item.theme.variables.themeMark))
+      .map(item => { return { value: item.name, title: item.title }; });
+    const availableThemesToItems = Object.keys(availableThemes)
+      .map(key => ({ id: key, value: availableThemes[key].value, title: availableThemes[key].title }));
+
+    if (availableThemesToItems.length > 1) {
+      this.changeThemeModel = new ListModel(
+        availableThemesToItems,
+        (item: any) => {
+          this.model.setTheme(item.value, themeMapper);
+          this.changeThemeAction.title = this.model.getCurrThemeTitle(themeMapper);
+          this.changeThemePopupModel.toggleVisibility();
+        },
+        true
+      );
+
+      this.changeThemePopupModel = new PopupModel(
+        "sv-list",
+        { model: this.changeThemeModel },
+        "bottom",
+        "center"
+      );
+
+      this.changeThemeAction = new Action({
+        id: "themeChanger",
+        iconName: "icon-theme",
+        component: "sv-action-bar-item-dropdown",
+        mode: "large",
+        title: getThemeTitle("defaultV2"),
+        needSeparator: true,
+        visible: <any>new ComputedUpdater<boolean>(() => {
+          const showSimulatorInTestSurveyTab = this.creator.showSimulatorInTestSurveyTab;
+          return this.creator.activeTab === "test" && showSimulatorInTestSurveyTab;
+        }),
+        action: () => {
+          this.changeThemePopupModel.toggleVisibility();
+        },
+        popupModel: this.changeThemePopupModel
+      });
+      this.changeThemePopupModel.onHide = () => { this.changeThemeAction.enabled = true; };
+      this.changeThemePopupModel.onShow = () => { this.changeThemeAction.enabled = false; };
```

Comment:
It is extra code. Pressed styles are being applied by default, when popup is opened, and action is not available for user.